xmipptomo/protocols/protocol_subtomo_map_back.py

```diff
@@ -26,16 +26,18 @@
 # *
 # **************************************************************************
 
-from pyworkflow.em import ALIGN_3D
-from pyworkflow.em.convert import ImageHandler
-import pyworkflow.em.metadata as md
-from pyworkflow.em.protocol import EMProtocol
+from os.path import basename
```

Comment:
This protocol is being testing with the users with real (and complex) data, it seems the code is fine, but maybe soon we find something to fix.

Reply:
This protocol is already working properly